tardis/plasma/properties/ion_population.py

```diff
@@ -243,3 +243,12 @@ def calculate(self, phi, partition_function, number_density):
             n_electron = 0.5 * (new_n_electron + n_electron)
 
         return ion_number_density
+
+class ElectronDensity(ProcessingPlasmaProperty):
+    name = 'electron_densities'
+
+    def calculate(self, ion_number_density):
+        ion_numbers = ion_number_density.index.get_level_values(1).values
+        ion_numbers = ion_numbers.reshape((ion_numbers.shape[0], 1))
+        n_electron = (ion_number_density.values * ion_numbers).sum(axis=0)
```

Comment:
@aoifeboyle it could be because of this, though I am not sure. Just check the dimensions of `ion_number_density` ?